provider/kubernetes/kubernetes.go

```diff
@@ -219,8 +220,13 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 					}
 				}
 
-				if rule := getRuleForPath(pa, i); rule != "" {
-					templateObjects.Frontends[baseName].Routes[pa.Path] = types.Route{
+				rule, err := getRuleForPath(pa, i)
+				if err != nil {
+					log.Errorf("fail to get rule from ingress path: %s", err)
+					continue NEXT_INGRESS
```

Comment:
How do you think about using a continue statement with label to skip the ingress?

Reply:
Hmm, this looks correct but I'm not entirely sure. I'd rather prefer to use our existing approach of a bare `continue`.

We need to invest some time into the Kubernetes provider in order to simplify our highly nested structure. Once we do that, I believe a more elegant way to skip Ingresses not relying on continuing the right for loop will emerge naturally.